<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileSettingsWindow</class>
 <widget class="QDialog" name="ProfileSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="columnCount">
          <number>1</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Create new profile.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resources.qrc">
             <normaloff>:/icons/profile-create.ico</normaloff>:/icons/profile-create.ico</iconset>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Duplicate the current profile.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resources.qrc">
             <normaloff>:/icons/profile-duplicate.ico</normaloff>:/icons/profile-duplicate.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Delete the current profile.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resources.qrc">
             <normaloff>:/icons/profile-delete.ico</normaloff>:/icons/profile-delete.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Load the selected profile.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources/resources.qrc">
             <normaloff>:/icons/profile-select.ico</normaloff>:/icons/profile-select.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Proile Information</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Profile Author</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditProfileAuthor"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditDescription"/>
         </item>
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="plainTextEditNotes"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Notes</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Game Studio</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditGameStudio"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditProfileName"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Profile Name</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Sound Mode</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_3">
           <property name="text">
            <string>Manual PCM volume</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_3">
           <property name="maximum">
            <number>15</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_4">
           <property name="text">
            <string>Manual reverb depth</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_5">
           <property name="text">
            <string>Manual PCM frequency</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_6">
           <property name="text">
            <string>Manual PCM channels</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_4">
           <property name="maximum">
            <number>127</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinBox_6">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBox_7">
           <property name="text">
            <string>Manual DAC config</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_2"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Game Tables</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="toolTip">
            <string>Manually specify songtable properties. Change with caution!</string>
           </property>
           <property name="title">
            <string>Songtable</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox">
              <property name="toolTip">
               <string>Enable to not use the automatic songtable scanner. Requires the songtable's position.</string>
              </property>
              <property name="text">
               <string>Manual songtable position</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="checkBox_2">
              <property name="toolTip">
               <string>Enable to not use the automatic songtable scanner. Requires the amount of songs the song table contains.</string>
              </property>
              <property name="text">
               <string>Manual song count</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="toolTip">
               <string>Specify which songtable to load if the game contains multiple. Only applicable for automatic songtable position.</string>
              </property>
              <property name="text">
               <string>Songtable index</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBox_2"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="toolTip">
            <string>Enable to manually specify the music player config. Change with caution!</string>
           </property>
           <property name="title">
            <string>Custom Playertable</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="3">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources/resources.qrc">
                <normaloff>:/icons/playlist-remove.ico</normaloff>:/icons/playlist-remove.ico</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources/resources.qrc">
                <normaloff>:/icons/playlist-add.ico</normaloff>:/icons/playlist-add.ico</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Player Count: 0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="4">
             <widget class="QTableWidget" name="tableWidget"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Profile Assignment</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="1">
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="toolTip">
            <string>Load this profile if the game matches a game code in the list.</string>
           </property>
           <property name="text">
            <string>Game Codes</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="toolTip">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="toolTip">
            <string>Magic bytes uniquely identify this profile. If agbplay finds theis byte string in the ROM, it will automatically load this profile instead of other profiles.</string>
           </property>
           <property name="text">
            <string>Magic Bytes</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProfileSettingsWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProfileSettingsWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
